README.adoc

```diff
@@ -1 +1,25 @@
+== Getting Started with Spring Cloud Task
+This guide walks you through an overview of Spring Cloud Task and the process of creating and launching short-lived
+batch application.
 
+=== What is Spring Cloud Task?
+A framework for building short-lived Spring Boot microservices for batch data processing. You can learn more about
```

Comment:
A framework for building short-lived Spring Boot microservices, such as batch data processing.

Reply:
The point is task is not just for batch processing.